File: WebApi/WebApi/Models/Notification.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Web;

using MongoDB;
using MongoDB.Bson;
using MongoDB.Driver;

namespace WebApi.Models
{
    public class Notification : Base
    {
        public Notification(Double latitude, Double longitude)
        {
            _t = "Notification";

            UserId = ObjectId.Empty;
            Name = "";
            DateTime = DateTime.UtcNow;
            PackageName = "";
            NotificationTicker = "";
            NotificationTitle = "";
            NotificationText = "";

            Location = new Location(UserId, ObjectId.Empty, "", latitude, longitude);
        }
        public ObjectId UserId { get; set; }
        public DateTime DateTime { get; set; }
        public string PackageName { get; set; }
        public string NotificationTicker { get; set; }
        public string NotificationTitle { get; set; }
        public string NotificationText { get; set; }
        public Location Location { get; set; }
    }
}

File: Apps/DriveSwitch/app/src/main/java/com/driveswitch/Notification.java
package com.driveswitch;

public class Notification
{
    public Notification()
    {
        UserId = "";
        PackageName = "";
        NotificationTicker = "";
        NotificationTitle = "";
        NotificationText = "";

        Location = new Location(UserId, Constants.EmptyObjectId, "", 0.00, 0.00);
    }

    public String UserId;
    public String PackageName;
    public String NotificationTicker;
    public String NotificationTitle;
    public String NotificationText;
    public Location Location;
}
